src/features/intake/pages/IntakeTemplatesPage.tsx

```
@@ -1053,6 +916,45 @@ function TemplateCard({
             </DropdownMenuContent>
           </DropdownMenu>
         </div>
+        {openEmbedDialog ? (
+          <Dialog
+            isOpen={openEmbedDialog}
+            onClose={() => setOpenEmbedDialog(false)}
+            title="Embed code"
+            description="Copy or inspect the embed snippet for this intake template."
+          >
+            <DialogBody>
+              <p className="mb-3 text-sm text-input-placeholder">Paste this script into your site's <code>&lt;head&gt;</code> or before <code>&lt;/body&gt;</code>.</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)